CurrencyDotNet.RestClient/CurrencyRestClient.cs

```diff
@@ -162,5 +157,30 @@ public void UpdatePosition()
         {
             throw new NotImplementedException();
         }
+        /// <summary>
+        /// Returns tickers for all the symbols for the last 24 hours
+        /// </summary>
+        /// <param name="cancellationToken"></param>
+        /// <returns></returns>
+        public async Task<CallResult<IEnumerable<Ticker>>> GetTickersAsync(
+            CancellationToken cancellationToken=default(CancellationToken))
+        {
+            var request = new GetTickerRequest(symbol: null);
```

Comment:
I think this is bad practice. Perhaps in the GetTickerRequest class you need to do the following:
`public GetTickerRequest(string? symbol = null)`

Reply:
Fixed.